android/assets/jsons/Tutorials.json

```diff
@@ -168,5 +168,53 @@
 "ⓨ: By the way, here's how an empire border looks like - it's in the national colours of the nation owning the territory.",
 "ⓩ: And this is the red targeting circle that led to the attack pane back under ⑬.",
 "What you don't see: The phone/tablet's back button will pop the question whether you wish to leave Unciv and go back to Real Life. On desktop versions, you can use the ESC key."
-  ]
+  ],
+    "Faith": [
+        "After building a shrine, you're civilization will start generating faith.",
+        "When enough faith has been generated, you will be able to found a pantheon.",
+        "A pantheon will provide a small bonus for your civilization that will apply to all your cities.",
+        "Each civilization can only choose a single pantheon belief, and each pantheon can only be chosen once.",
+        "Generating more faith will allow you to found a religion."
+    ],
+    "Religion": [
+        "When you have gathered enough religion, a great prophet will be born in one of your cities.",
+        "This great prophet can be used for multiple things: Constructing a holy city, founding a religion and spreading your religion.",
+        "When founding your religion, you may choose another two beliefs. The founder belief will only apply to you, while the follower belief will apply to all cities following your religion.",
+        "Additionally, the city where you used your great prophet will become the holy city of that religion.",
+        "If after founding your religion another great prophet is born in one of your cities, you can use him to enhance your religion.",
```

Comment:
"If, after founding your religion," - commas or restructure entirely ... maybe "Once your Religion is founded, it can be enhanced by another GP" then maybe hint at "birthing" rate formula? Dunno.